### ISSUE TITLE
Side Navigation Broken with 2.40.2

### ISSUE THREAD

**Issue by xxcuratorxx** [dotnet/docfx#3606]
**DocFX Version Used**: 2.40.2

**Template used**: (`default` or `statictoc` or contain custom template) statictoc

**Steps to Reproduce**:

1. docfx init
2. repalce toc.yml content in articles folder by
```
- name: Sub Introduction
- name: Introduction
  items:
  - name: Introduction 1
    href: intro.md
  - name: Introduction 2
    href: intro.md
  - name: Introduction 3
    href: intro.md

```
3. docfx --template statictoc

**Expected Behavior**:
Nice Side Navigation on Articles page

**Actual Behavior**:
Strange Scrollbar and layout of items in navigation

Works with 2.39.2 or 2.40.2 with default template

**Comment by superyyrrzz** [comment]
From the sample in #3536 , try add ` dropdown: true`:

```yml
- name: Sub Introduction
- name: Introduction
  dropdown: true
  items:
  - name: Introduction 1
    href: intro.md
  - name: Introduction 2
    href: intro.md
  - name: Introduction 3
    href: intro.md
```

**Comment by xxcuratorxx** [comment]
Hi,
just to make it clear and avoid confusion: I am NOT talking about the navbar in the header. My problem in with the side navigation (left hand side)

However, I added the dropdown: true line, which leads a triangle besides the item (to visualize dropdown), but doesn't work on click (remains closed). I personally think that the +/- style of former versions was working as expected but somehow broken

**Comment by superyyrrzz** [comment]
@xxcuratorxx Thank you for the detailed information. I will look into this.

**Comment by superyyrrzz** [comment]
fixed in 2.40.3: https://github.com/dotnet/docfx/releases/tag/v2.40.3
